tests/test-files/test-ssh-buildx.yaml

```diff
@@ -0,0 +1,19 @@
+# Please note that for the SSH unit tests to pass on a local machine, the private key needs to be placed in ~/.ssh/buildrunner-deploy-id_rsa
+# Look in the .github/workflows/build.yaml file for the SSH private key
+use-legacy-builder: False
```

Comment:
random, but `False` is pythonic, but `false` is "more correct" for YAML (though both work I believe).

Reply:
Good to know. I wasn't aware that YAML had a preference. `False` definitely works here.